### ISSUE TITLE
Gossip: an online presentation tool inspired by impress.js

### ISSUE THREAD

**Issue by pearmini** [impress/impress.js#780]
Thanks a lot for your guys to create impress.js, I like it very much 😘! 

But during authoring presentation by impress.js, I found it difficult to make layout and choose the proper switch animation for each slides, so I use the **word cloud algorithm** to place each slides automatically and apply switch animation depend on the relationship between two slides before and after. 

Furthermore, I also found a series of problem related to author and delivery presentation, so I create an online user interface called **Gossip** to solve them. [[link](https://pearmini.github.io/gossip/)][[github](https://github.com/pearmini/gossip)]

I hope your guys like Gossip and give me some suggestions to improve it!

![author](https://github.com/pearmini/gossip/raw/master/screenshots/edit_mode.png)
![delivery](https://github.com/pearmini/gossip/raw/master/screenshots/screening_mode.png)

**Comment by henrikingo** [comment]
Thanks for sharing! Looks like an interesting approach.

If you want, you're welcome to add a link to your tool in the wiki: https://github.com/impress/impress.js/wiki/Examples-and-demos#authoring-tools

I have also authored a GUI authoring tool: https://github.com/henrikingo/impressionist But since I took over maintainership of impress.js itself, I haven't really had time to continue on Impressionist. So the space is still open for a great impress.js visual authoring tool!
